test/test-eth-ledger-bridge-keyring.js

```diff
@@ -47,10 +49,19 @@ chai.use(spies)
 describe('LedgerBridgeKeyring', function () {
 
   let keyring
+  let sandbox
 
-  beforeEach(function () {
+  beforeEach(async function () {
+    sandbox = chai.spy.sandbox()
     keyring = new LedgerBridgeKeyring()
     keyring.hdk = fakeHdKey
+    keyring.setAccountToUnlock(fakeAccounts[0])
+    sandbox.on(keyring, 'unlock', (_) => Promise.resolve(fakeAccounts[0]))
+    await keyring.addAccounts()
```

Comment:
Nit: Is this step relevant to all of the tests? Generally I've been trying to reduce the amount of stuff done in the `beforeEach`, as it makes it difficult to understand what each individual tests needs.

Reply:
It was relevant to about 10 of the thirty tests, so I extracted it into a helper method that can be called in the relevant tests. I then returned the tests that were modified due to this change back to their original state.

Reply:
The helper function looks great! Much easier to read.